dali/operators/geometry/affine_transforms/transform_translation.cc

```diff
@@ -35,6 +35,25 @@ The number of dimensions of the transform is inferred from this argument.)code",
   .NumOutput(1)
   .AddParent("TransformAttr");
 
+DALI_SCHEMA(TransformTranslation)
+  .DocStr(R"code(Produces a translation affine transform matrix.
```

Comment:
I'd remove the documentation string and replace it by a warning saying: "TransformTranslation" has been renamed, use transforms.Translation instead."